helm/libero-reviewer/templates/deployment-submission.yaml

```diff
@@ -1,30 +1,30 @@
 apiVersion: apps/v1beta2
 kind: Deployment
 metadata:
-  name: {{ include "libero-reviewer.fullname" . }}-server
+  name: {{ include "libero-reviewer.fullname" . }}-submission
   labels:
     app.kubernetes.io/name: {{ include "libero-reviewer.name" . }}
     helm.sh/chart: {{ include "libero-reviewer.chart" . }}
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
-    app.kubernetes.io/component: server
+    app.kubernetes.io/component: submission
```

Comment:
Surely this should be qualified with `reviewer` ?

Reply:
From my understanding its a component of the chart (which is libero-reviewer)

Reply:
The scope of `app.kubernetes.io/component` (at least the way we use it) is to distinguish between different Deployments and Services within the same release of the chart. You could call them `a`, `b`, `c` at the bare minimum as it doesn't need to match anything else than the [`Service` `spec.selector` field](https://github.com/elifesciences/libero-reviewer-formula/blob/master/helm/libero-reviewer/templates/service-server.yaml#L21).

Since this Deployment is managing Pods with a single container, it makes sense to use the project name for that container. Even if there were multiple containers in the Pod, there would probably be a primary one. So match the `.Values.images` key being used.

Reply:
TL;DR: yes, but should use `reviewer(-|_)submission` consistently everywhere then.

Reply:
Including [`client` which doesn't](https://github.com/elifesciences/libero-reviewer-formula/blob/master/helm/libero-reviewer/templates/deployment-client.yaml#L17).

Reply:
So, overkill?

Reply:
probably